ui-v2/app/styles/components/service-list.scss

```diff
@@ -0,0 +1,6 @@
+@import './service-list/index';
+
+.service-list li {
+  @extend %composite-row;
+  @extend %service-list;
+}
```

Comment:
I'm quibbling a bit over the name of this here. The `li` would be a `-row` of some sort, not a `-list`.

So either:

```css
.consul-service-list {
  @extend %consul-service-list;
}
%consul-service-list li {
  @extend %composite-row;
}
```

or

```css
%consul-service-list li {
  @extend %consul-service-row;
}
%consul-service-row {
  @extend %composite-row;
}
```

I'd be tempted to go with the latter as it looks like we are always talking about `-row`s with CSS not lists. It also covers us for if we ever need to mix different types of rows in the same list. What do you think?